pyproject.toml

```
@@ -0,0 +1,123 @@
+[build-system]
+requires = ["setuptools >= 61.0", "setuptools-scm>=8"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pdfminer.six"
+dynamic = ["version"]
+description = "PDF parser and analyzer"
+readme = "README.md"
+license = "MIT"
+requires-python = ">=3.8"
+authors = [
+    { name = "Yusuke Shinyama + Philippe Guglielmetti", email = "pdfminer@goulu.net" },
```

```suggestion
    { name = "Yusuke Shinyama" },
    { name = "Philippe Guglielmetti", email = "pdfminer@goulu.net" },
    { name = "Pieter Marsman" },
```

What do you think of using 1 dict per author? 

And I think it is time to add my name as well.

---

Oh, yes, indeed, it should be one dict per author and as maintainer you should be there too...